src/main/java/Nyx.java

```diff
@@ -0,0 +1,65 @@
+import nyx.Ui;
+import nyx.Parser;
+import nyx.Storage;
+import nyx.TaskList;
+import nyx.NyxException;
+
+/**
+ * Entry point of the Nyx chatbot application.
+ */
+public class Nyx {
+    private static final String logo = " __      _\n"
+            + "|   \\   | |__    __ __     __\n"
+            + "| |\\ \\  | |\\ \\  / / \\ \\   / /\n"
+            + "| | \\ \\ | | \\ \\/ /   \\ \\ / / \n"
+            + "| |  \\ \\| |  \\  /    /    /\n"
+            + "| |   \\   |  / /    / / \\ \\\n"
+            + "|_|    \\__| /_/    /_/   \\_\\\n";
+    private final Ui ui;
+    private TaskList taskList;
+    private final Storage storage;
```

Comment:
super small thing, but perhaps can grp the variables with the similar prefix together! in this case that would be:
```suggestion
    private final Ui ui;
    private final Storage storage;
    private TaskList taskList;
```